### PR TITLE
Revision

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
- floating action buttons
- disabled buttons
- fixed success and error handler input box
- added shadow and outdated card
- added stacked notificaton list
- fixed the alert buttons functionality

image
image
image
image
image
image

### CODE REVIEW

File: CSS/button.css [hunk 1]
```diff
@@ -23,4 +23,59 @@
...
+
+#contained-btn-red {
+    border-radius: 1rem;
+    color: var(--red);
+}
+
+#contained-btn-red:hover {
+    background-color: var(--red);
+    color: var(--white);
+}
```
Comment (JantuDeb) [inline CSS/button.css:64]: avoid using ids

**Review by JantuDeb — APPROVED** [review]
These components looks really awesome

**Comment by ajit-sharmaGH** [comment]
> These components looks really awesome

Thank you @JantuDeb  !

**Review by SaurabhChawda — APPROVED** [review]
code and design look good to me but you can change the dimension of the home page image.

**Pull request merged**